CAFAna/Core/SpectrumLoader.cxx

```diff
@@ -22,8 +24,10 @@
 #include <cmath>
 #include <iostream>
 
+#include "TChain.h"
 #include "TFile.h"
 #include "TH2.h"
+#include "TRandom3.h"
```

Comment:
I don't think we can need TChain or TRandom3 anymore.

Reply:
Yes, they're almost certainly not needed.